### ISSUE TITLE
Need Help！

### ISSUE THREAD

**Issue by kmz0110** [DevLARLEY/wingfox-downloader#9]
Thank you,Great God, how can I communicate with you in real time, there are many questions for you

**Comment by DevLARLEY** [comment]
this encryption technique is only used by polyv which can be implemented on any website. To adjust the script for other websites you should only have to change the API endpoints a bit to get it working

**Comment by kmz0110** [comment]
Thank you,Great God, how can I communicate with you in real time, there are many questions for you

**Comment by kmz0110** [comment]
Is the Great God awake?

**Comment by kmz0110** [comment]
I changed the script you wrote from the command line input to the Python code, and entered the vid and other platform custom tokens, and successfully downloaded the video, which proves that the other platforms you mentioned earlier are also feasible, which is really cool. I still have a doubt, that is, the call Js download is too slow, can the cracking process be directly written as a python algorithm, and is the 16-bit key cracked in the V13 version time-sensitive? Because the previous lower version was not pdx, my storage resources were directly stored m3u8 files and the Base64 encoded key of the cracked key, and then downloaded from the platform when needed, which made the file seem small. Very convenient, I don't know if you understand what I mean? Finally, thank you very much for your hard work and contribution, thank you!

**Comment by DevLARLEY** [comment]
PolyV uses many different encryption methods between videos so it's pretty much random which one you're gonna get. The can range from "simple" ts header encryption to a completely new demuxer written in WASM with a custom video file format. N_m3u8DL-RE is already taking care of the header encryption so only the demuxing part is written in JS. It would be way to much work to convert all of that JS code to Python and it would actually make it slower because Python is slower than WASM.

**Comment by kmz0110** [comment]
Do you have any contact information? Do you mind chatting with you in real-time?

**Comment by DevLARLEY** [comment]
larley on discord

**Comment by kmz0110** [comment]
Great God, Polyv version V13, I now use your JS crack, reported this error prompt: "20:27:38.961 INFO : Start downloading... Vid Kbps
                                                                     
20:27:41.999 WARN: The referenced object type does not support the operation attempted. (hw-dts.videocc.net:443)
20:27:42.001 ERROR: Download first segment failed!", do you check if you can still crack the version, or am I not getting something right? Because it was okay a few days ago, but now suddenly it doesn't

**Comment by kmz0110** [comment]
Great God, Polyv version V13, I now use your JS crack, reported this error prompt

seed_const => 214
hlsLevel => app
hlsPrivate => 2
22:30:27.015 INFO : 输出被重定向, 将清除ANSI颜色
22:30:27.038 INFO : N_m3u8DL-RE (Beta version) 20240630
22:30:27.039 INFO : 加载URL: 2b538992553206a2b7162ce3ad0a7cf5_2.m3u8
22:30:27.047 INFO : 内容匹配: HTTP Live Streaming
22:30:27.048 INFO : 正在解析媒体信息...
22:30:27.061 WARN : 写出meta json
22:30:27.062 INFO : 已解析, 共计 1 条媒体流, 基本流 1 条, 可选音频流 0 条, 可选字幕流 0 条
22:30:27.062 INFO : Vid Kbps | 146 Segments | ~24m15s
22:30:27.062 INFO : 已选择的流:
22:30:27.062 INFO : Vid Kbps | 146 Segments | ~24m15s
22:30:27.062 WARN : 写出meta json
22:30:27.063 INFO : 保存文件名: 2b538992553206a2b7162ce3ad0a7cf5_2
22:30:27.064 INFO : 开始下载...Vid Kbps
                                                                      




22:30:30.118 WARN : 参考的对象类型不支持尝试的操作。 (hw-dts.videocc.net:443)
22:30:30.119 ERROR: Download first segment failed!
[ERROR]: Unable to request key (403): 

Process finished with exit code 1

**Comment by DevLARLEY** [comment]
Can you post your video id and cookie?

**Comment by kmz0110** [comment]
Of course, I'd love to, please send me the discord link, I want to communicate with you in real time

**Comment by DevLARLEY** [comment]
larley on Discord

**Comment by kmz0110** [comment]
> Discord 上的 larley
Because I'm new to Discoord, I don't know how to find you, I'm Discoord ID: amo0126, nicknamed Amo, why don't you please add me?

**Comment by kmz0110** [comment]
I've already added you, please agree
